packages/devtools_app/lib/src/utils.dart

```diff
@@ -1223,6 +1223,15 @@ class ListValueNotifier<T> extends ChangeNotifier
     _listChanged();
   }
 
+  /// Replaces all elements in the list and notifies listeners. It's preferred
+  /// to calling .clear() then .addAll(), because it only notifies listeners
+  /// once.
+  void replaceAll(Iterable<T> elements) {
+    _rawList.clear();
```

Comment:
let's use `_rawList = <T>[];` here to be consistent with how the `clear` method clears out the list

Reply:
Good point, done!